posts/views.py

```diff
@@ -69,6 +81,13 @@ def post_create(request):
         save_hashtags(parsedString['hashtags'], post)
     return redirect(request.META['HTTP_REFERER'] or 'landingpage')
 
+def set_post_replies(post):
```

Comment:
indeed, i forgot to take that out and put it into `set_post_extra` and the bug it's already fixed, just finishing another couple of things, i'll get back to this in the next 30 min